numpy/_core/src/umath/ufunc_object.c

```diff
@@ -4363,6 +4363,20 @@ ufunc_generic_fastcall(PyUFuncObject *ufunc,
             Py_INCREF(tmp);
             PyTuple_SET_ITEM(full_args.out, i-nin, tmp);
         }
+
+        /* Extra positional args but *no* keywords */
+        if ((ufunc == UFUNC_MAXIMUM || ufunc == UFUNC_MINIMUM) && !all_none) {
```

Comment:
This is a bit tautological, you already have both `ufunc ==` checks just below.  TBH, I think it would be fine to just make one path though and don't care about the duplication.  (or use `ufunc->name`.)